README.md

```diff
@@ -10,4 +10,9 @@ To install **kns** you can just run this command:
 ```bash
 curl https://raw.githubusercontent.com/blendle/kns/master/kns -o /usr/local/bin/kns && chmod +x $_
 ```
+Or use our homebrew tap:
+```bash
+$ brew tap blendle/blendle
+$ brew install kns
+```
```

Comment:
I've been adding `blendle/blendle/x` to `script/bootstrap` output, but I'm fine with either of them 👍 

Also, minor nit: you should see about using a markdown linter, using newlines around code-blocks and headers makes the text more readable in a non-formatted way 👍